pypopquiz/video.py

```diff
@@ -64,8 +70,13 @@ def create_video(kind: str, round_id: int, question: Dict, question_id: int, out
     if file_name.exists():
         file_name.unlink()  # deletes a previous version
 
-    stream = VideoBackend(video_file)  # currently hard-coded since FFMpeg is the only back-end
-    stream = filter_stream(stream, kind, round_id, question, question_id, width, height)
-    stream.run(file_name)
+    if backend == 'ffmpeg':
+        be = ppq.backends.ffmpeg.FFMpeg
+    else:
```

Comment:
Now that we have a string for backend selection I would make this an `elif backend == 'moviepy' and add an `else` raising an exception to indicate an incorrect backend.

Reply:
Will edit.